docs/meshstack.how-to.create-customer.md

```diff
@@ -0,0 +1,30 @@
+---
+id: meshstack.how-to.create-customer
+title: How to create a meshCustomer via API
+---
+
+This how-to guide is targeted at Cloud Foundation Partners. You need an meshObject API user to run through the steps below.
+
+## Step 1: Install Postman
+
+First install postman. While any tool will do, in this how-to will use [Postman](https://www.postman.com/).
+
+## Step 2: Import existing Customer Owners into the meshObjectCollection
+
+Import the existing [meshCustomerUserBindings](https://docs.meshcloud.io/api/#_meshcustomeruserbinding) for the Customer Owners into the meshObjectCollection.
```

Comment:
q: Can these also be realtive even though they link to the api docs?

Reply:
I tried it out locally by using [meshCustomerUserBindings](../api/#_meshcustomeruserbinding). The resulting URL would be the API docs. However,`yarn lint` gives me `error  .../meshcloud-docs/api/#_meshcustomeruserbinding is dead. (ENOENT: no such file or directory, access '/Users/fzieger/meshcloud/meshcloud-docs/api/')        no-dead-link
`

So there might be a way to use relative links, but I am not sure how to do that.